torch/onnx/_fx.py

```diff
@@ -0,0 +1,330 @@
+# Owner(s): ["module: onnx"]
```

Comment:
I would put this file in either `_internal` or the `experimental` path. @BowenBao do you have suggestions?

Reply:
It will be in `_internal`.